packages/zk-certificates/package.json

```diff
@@ -44,16 +44,23 @@
     "circom2": "^0.2.13",
     "circomlib": "^2.0.5",
     "circomlibjs": "^0.1.7",
+    "csvtojson": "^2.0.10",
     "ethers": "^5.7.2",
     "ffjavascript": "^0.3.0",
+    "i": "^0.3.7",
     "keccak256": "^1.0.6",
     "snarkjs": "^0.5.0"
   },
   "devDependencies": {
     "@metamask/eth-sig-util": "^7.0.1",
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.4",
+    "@nomicfoundation/hardhat-ethers": "^3.0.6",
```

Comment:
I like the upgrade, but it might cause conflicts or confusion if both `"@nomicfoundation/hardhat-ethers": "^3.0.6"` and `"@nomiclabs/hardhat-ethers": "^2.2.1"` are available.

Reply:
I also started this some time ago, but never finished it because it requires a couple migration changes: https://github.com/Galactica-corp/galactica-monorepo/compare/main...hardhat-update-verifier

Reply:
I created a task in Asana for it because it makes sense to upgrade it anyway: https://app.asana.com/0/1206463951101181/1208463889193907/f